### DISCUSSION TITLE
How to Improve quality of output with higher definition, more detailed outfits?

### DISCUSSION THREAD

**Discussion by oldhand7** [lllyasviel/Fooocus#2908] [Q&A] [answered]
Some of the generated images are bad quality and the asian females are not generated well.
Does it require training an model?
Please let me know. 

Thank you.

**Comment by mashb1t** [comment, marked as answer]
@oldhand7 you've asked this already in https://github.com/lllyasviel/Fooocus/discussions/2906#discussioncomment-9414507.
Please don't spam new discussions or you will get banned.

**Reply by oldhand7** [reply]
Ok.

**Reply by 841372** [reply]
Send me video please

**Reply by mashb1t** [reply]
@841372 https://www.youtube.com/watch?v=dQw4w9WgXcQ
